### ISSUE TITLE
Missing "Schema"-property in "TYPO3.Flow.security.schema.yaml" for "token"-property

### ISSUE THREAD

**Issue by dnepro** [neos/flow-development-collection#700]
On running "flow configuration:validate" i receive an error, stating that: 
```yaml
- Settings.TYPO3.Flow.security.authentication.providers.Flowpack.Neos.FrontendLogin:Frontend.token -> This property is not allowed here, check the spelling if you think it belongs here.
```
This is due to this "Settings"-configuration:
```yaml
TYPO3:
  Flow:
    security:
      authentication:
        providers:
          'Flowpack.Neos.FrontendLogin:Frontend':
            provider: 'X\Y\FrontendLogin\Authentication\Provider\PersistedUsernamePasswordWlkidProvider'
            token: 'X\Y\FrontendLogin\Authentication\Token\UsernamePasswordWlkid'
            requestPatterns:
              'Flowpack\Neos\FrontendLogin\Security\NeosRequestPattern': 'frontend'
```
While for me this is completely valid and functional and can be found in the [documentation](http://flowframework.readthedocs.io/en/stable/TheDefinitiveGuide/PartIII/Security.html#reuse-of-tokens-and-providers).

And after asking on the [Flow Slack Channel](https://neos-project.slack.com/archives/flow-general/p1478621434002367) i would like to create a PR, adding:

```yaml
  'authentication':
    properties:
      'providers':
        additionalProperties:
          properties:
            'token': { type: string }
```
If this is a valid PR i would just like to know against which branch to do the PR, master?

**Comment by kdambekalns** [comment]
This should be done against the lowest branch that actually supports `token` at that place in the settings.

**Comment by dnepro** [comment]
Ok so i just looked up the documentation and it seems this was introduced with 3.0, because in 2.X it was called "tokenClass". So i will do the PR against the 3.0 branch, thanks Karsten!

**Comment by bwaidelich** [comment]
I'm a little bit puzzled:
So apparently this has been (re)introduced with 1dc4a5491b7415843fa62ff66921836ac2341149 by @kdambekalns to almost all branches a long time ago..
#130 fixes the schema, but only for 3.3.0+

**Comment by dnepro** [comment]
Ahh i [see](https://github.com/neos/flow-development-collection/pull/130/files?diff=split#diff-34b7ac8ce37378815adbc8a441fb7302R45)..
but as Karsten said, this should be done for the lowest branch supporting `token`, which in my case is 3.0 but in general you're right. But i don't know what to do with this issue.

**Comment by bwaidelich** [comment]
Well, if it's broken for 3.0-3.2 it should be fixed for those branches as in: apply to 3.0. I was just a bit confused

**Comment by dnepro** [comment]
Ahh ok now i understand you're confusion, yes definitely
